edx_rest_framework_extensions/auth/jwt/tests/test_middleware.py

```diff
@@ -276,15 +276,15 @@ def setUp(self):
         ('/loginredirectifunauthenticated/', True, 200),
         ('/isauthenticatedandloginredirect/', False, 302),
         ('/isauthenticatedandloginredirect/', True, 200),
-        ('/isauthenticated/', False, 403),
-        ('/isauthenticated/', True, 403),
+        ('/isauthenticated/', False, 401),
+        ('/isauthenticated/', True, 401),
```

Comment:
Do you know which specific upgrade triggered this?  Just wondering if it'll be a problem for the mobile apps, or if we already covered that case with the oauthlib upgrade testing.

Reply:
Im not sure. I think this might be because of the way tests are configured.